duckduckgo/Android
# Feature/google analytics surrogates

Asana Issue URL: https://app.asana.com/0/488551667048375/518654221411512

## Description
Implement Google Analytics surrogates to prevent some sites from breaking


## Steps to Test this PR:
1. Visit https://raspberrypi.org/
1. Click "Buy a raspberry pi"
1. Click "Buy now"
1. Choose "element14" 

If the site successfully loads, this is a good indicator the surrogate is working. For comparison, do the same in `develop` branch and confirm it doesn't work there.
